### ISSUE TITLE
Joysticks all black?

### ISSUE THREAD

**Issue by silverl** [kpshek/ledblinky-controls-colors#8]
Just discovered your wonderful repo. I'm building my first cab with a Pi4b, RetroPie and LEDSpicer.

Things are working fine except I noticed that a great many of the games have joysticks colored "black", which at least for my RGB controller means "off". I splurged for the RGB upgrades for my Ultimarc ServoStiks, so I noticed.

Just wondering if I'm missing something? I know I can just edit the file and enable colors on the joysticks, but I guess I was surprised to see so many colorless joystickss in the colors.ini file.

Thanks for your time.

**Comment by kpshek** [comment]
@silverl - I hadn't noticed that as I don't have LEDs in my joysticks. Out of curiosity, what color will you thinking the joysticks would be?

For games where a given player's buttons are all a single color (eg, TMNT), I imagine the joystick color will match. But for games where the button colors don't match, I'm guessing you'd go with something like white?

**Comment by silverl** [comment]
Yes, that's exactly what I was thinking: white joystick if the buttons are mixed. 

Although if there's some kind of dominant color for a character, I'd probably try to color the joystick that. Can't think of a game like that offhand though.

Thanks for replying!

**Comment by kpshek** [comment]
No problem - if you want to submit a PR for this please feel free to and I'll get it merged in!
